Crash report 3.9.1: AssertionError in StatsCommentsFragment

I've checked all reports, it an Android 4.2.2 bug only. Probably a platform bug. I tried to repro on a 4.2.2 emulator with en-gb or en-au selected but it didn't crashed.

Also it's weird because the crash occurs on a `Fragment.getString(STR, number)` call only in `StatsCommentsFragment`, when it's also used other stats fragment (`StatsFollowersFragment` for example).

It's rare (8 users affected), but I suspect it's super annoying (crash loop on app start).

```
java.lang.AssertionError: couldn't initialize LocaleData for locale en-gb
       at libcore.icu.LocaleData.initLocaleData(LocaleData.java:167)
       at libcore.icu.LocaleData.get(LocaleData.java:121)
       at java.util.Formatter.format(Formatter.java:1039)
       at java.util.Formatter.format(Formatter.java:1009)
       at java.lang.String.format(String.java:1988)
       at android.content.res.Resources.getString(Resources.java:445)
       at android.app.Fragment.getString(Fragment.java:801)
       at org.wordpress.android.ui.stats.StatsCommentsFragment.updateUI(StatsCommentsFragment.java:92)
       at org.wordpress.android.ui.stats.StatsAbstractListFragment$3.onReceive(StatsAbstractListFragment.java:456)
       at android.support.v4.content.LocalBroadcastManager.executePendingBroadcasts(LocalBroadcastManager.java:297)
       at android.support.v4.content.LocalBroadcastManager.access$000(LocalBroadcastManager.java:46)
       at android.support.v4.content.LocalBroadcastManager$1.handleMessage(LocalBroadcastManager.java:116)
       at android.os.Handler.dispatchMessage(Handler.java:107)
       at android.os.Looper.loop(Looper.java:194)
       at android.app.ActivityThread.main(ActivityThread.java:5422)
       at java.lang.reflect.Method.invokeNative(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:525)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:837)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:604)
       at dalvik.system.NativeStart.main(NativeStart.java)
```

--- a/WordPress/src/main/java/org/wordpress/android/ui/stats/StatsCommentsFragment.java
+++ b/WordPress/src/main/java/org/wordpress/android/ui/stats/StatsCommentsFragment.java
@@ -89,12 +89,14 @@ public class StatsCommentsFragment extends StatsAbstractListFragment {
         if (mDatamodels[1] != null && !isErrorResponse(1)) { // check if comment-followers is already here
             mTotalsLabel.setVisibility(View.VISIBLE);
             int totalNumberOfFollowers = ((CommentFollowersModel) mDatamodels[1]).getTotal();
-            mTotalsLabel.setText(
-                    getString(
-                            R.string.stats_comments_total_comments_followers,
-                            FormatUtils.formatDecimal(totalNumberOfFollowers)
-                    )
-            );
+            try {
+                mTotalsLabel.setText(getString(R.string.stats_comments_total_comments_followers,
+                                     FormatUtils.formatDecimal(totalNumberOfFollowers)));
+            } catch (AssertionError e) {
+                // Workaround for a weird bug in Android 4.2.2 with en_GB or en_AU locale
+                // https://github.com/wordpress-mobile/WordPress-Android/issues/2639
+                mTotalsLabel.setVisibility(View.GONE);
+            }
         } else {
             mTotalsLabel.setVisibility(View.GONE);
         }

